src/serveses/move-search/MovieSearchService.js

```diff
@@ -0,0 +1,17 @@
+import {filmModels} from "../../models/FilmModel";
+
+export class MovieSearchService {
+  static filterFilms(searchBy, searchValue) {
+    if (searchBy === 'genre') {
+      return filmModels.filter(film => film.genre === searchValue);
+    }
+    if (searchBy === 'title') {
+      return filmModels.filter(film => film.title === searchValue);
```

Comment:
Can we somehow avoid code duplication? For ex. in the following way: `return filmModel.filter(film => film[searchBy] === searchValue)` ?